projects/ngx-prx-styleguide/browserslist

```diff
@@ -0,0 +1,11 @@
+# This file is currently used by autoprefixer to adjust CSS to support the below specified browsers
+# For additional information regarding the format and rule options, please see:
+# https://github.com/browserslist/browserslist#queries
+#
+# For IE 9-11 support, please remove 'not' from the last line of the file and adjust as needed
+
+> 0.5%
+last 2 versions
+Firefox ESR
+not dead
+IE 9-11
```

Comment:
I'm fairly sure our IE support is limited to 11 (or maybe 10?)

@sandikbarr could you weigh in?

Reply:
yes, just IE 11